citusdata-singlenode.yml

```diff
@@ -0,0 +1,45 @@
+# This stack script installs citusdata in as single node cluster 
+# 3 instances will be created: master, worker1 and worker2
+
+provider:
+  aws:
+    access_key: '${var.aws_access_key}'
+    secret_key: '${var.aws_secret_key}'
+resource:
+  aws_instance:
+    citus-single:
+      instance_type: t2.medium
+      ami: ''
+      tags:
+        Name: '${var.koding_user_username}-${var.koding_group_slug}'
+      user_data: |-
+        set -e
```

Comment:
![image](https://cloud.githubusercontent.com/assets/20778330/22350922/a5317784-e3e3-11e6-8c6f-5ff0b2bdac6f.png)
still the error:
![image](https://cloud.githubusercontent.com/assets/20778330/22350999/00cbc428-e3e4-11e6-824e-95831c8a8f44.png)